packages/client/src/components/Modal/Modal.component.js

```diff
@@ -1,10 +1,17 @@
-import React from 'react';
 import PropTypes from 'prop-types';
+import React, { useEffect } from 'react';
 import './modal.styles.css';
 
 const Modal = ({ toggle, open, title, children }) => {
+  useEffect(() => {
+    if (open) {
+      document.body.style.overflow = 'hidden';
+    } else {
+      document.body.style.overflow = '';
+    }
```

Comment:
I would also include the cleanup function to handle the component being unmounted.

Reply:
Skipped it because linter is cursing 🙈